src/ir/headers/mantle.hpp

```diff
@@ -147,10 +149,13 @@ Namespace* CoreIRLoadHeader_mantle(Context* c) {
     def->connect("self.in", "self.out");
   });
 
-  Params counterParams({{"width", c->Int()},
-                        {"has_en", c->Bool()},
-                        {"has_srst", c->Bool()},
-                        {"has_max", c->Bool()}});
+  wire->setPrimitiveExpressionLambda([]() { return vAST::make_id("in"); });
```

Comment:
Most of the changes in this file are auto style changes from clang format, except this line which adds the verilogAST primitive support for mantle wire.